admin/client/src/containers/CampaignAdmin/CampaignAdmin.js

```diff
@@ -255,7 +255,12 @@ class CampaignAdmin extends SilverStripeComponent {
   }
 
   addCampaign() {
-    // Add campaign
+    const path = this.props.sectionConfig.campaignViewRoute
+      .replace(/:type\?/, 'set')
+      .replace(/:id\?/, 0)
```

Comment:
Adding a `0` for a creation URL is pretty ugly. Also, the routing doesn't work when going to `/admin/campaigns/set/0/create` directly.